neoload/neoload_cli_lib/tools.py

```diff
@@ -90,12 +90,18 @@ def get_named_or_id(name, is_id_, resolver):
     return rest_crud.get(endpoint + "/" + name)
 
 
-def ls(name, is_id_, resolver):
+def ls(name, is_id_, resolver, filter_spec=None, input_params=None):
```

Comment:
input_params is actually the list of query params that are read by the API.
this variable could have a better name like "allowed_api_query_params" (also in filtering.py)

Reply:
Resolved in 94e75c2